### PR TITLE
[A3] Add Google Lighthouse as a new milestone

### PR DESCRIPTION
**Pull request by ianngiaw**
Title is self-describing again. Converted milestones 9 and 10 to 9a and 9b, shifted milestones > 10 up by 1.

### CODE REVIEW

File: coursework/mobile/index.md [hunk 1]
```diff
@@ -915,7 +915,7 @@ objects, one workaround is to store the object in stringified-JSON format.
 For a complete list of Web Storage's capability, visit <http://dev.w3.org/html5/webstorage/>.
 
 <div class="box">
-  <strong class="milestone-counter">Milestone 9:</strong> Implement and briefly describe
+  <strong class="milestone-counter">Milestone 9a:</strong> Implement and briefly describe
```
Comment (yangshun) [inline coursework/mobile/index.md]: Let's just merge into one (can directly merge or reduce some scope). Our grading spreadsheet doesn't handle sub milestones well I think lolol.

File: coursework/mobile/index.md [hunk 2]
```diff
@@ -1406,12 +1435,14 @@ contains the following:
     your answers to all compulsory milestones that require
     written answers.
 
-3.  A one/two-page pitch of your application,
+3.  Lighthouse `html` report, `group-<number>-lighthouse.html`.
```
Comment (yangshun) [inline coursework/mobile/index.md]: 👍

Comment (yangshun) [reply coursework/mobile/index.md]: While you're at this, can help change all the bullet points to `1.`? They'll increment automatically and if we insert in the middle we don't have to update the rest.

**Review by yangshun — APPROVED** [review]
Just some minor comments. :shipit: yourself when you're done!

**Comment by ianngiaw** [comment]
@yangshun

**Pull request merged**